frontend/src/bundles/talent-onboarding/components/step-content/styles.module.scss

```diff
@@ -102,10 +102,7 @@
     }
 
     .buttonPlaceholder {
```

Comment:
you can just remove button from template

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/78347072/cb5a24d1-6deb-4131-8afd-ecf7e8614535)

Reply:
fixed